src/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -136,8 +136,11 @@ public void createDataSource() throws SQLException {
               props.put("dataSource.sslcert", workConf.getSslCert());
               props.put("dataSource.sslkey", workConf.getSslKey());
             }
-
+            
             HikariConfig config = new HikariConfig(props);
+            if (workConf.getJdbcURL().length()>0) {
+              config.setJdbcUrl(workConf.getJdbcURL());
+            }
```

Comment:
does it matter? This would be a user input problem. if the user sets inconsistent properties, they'd get an error.
Based on my understanding of the hikari code, it looks to me that the properties set via code override the ones already in the jdbcUrl connection string.

Reply:
If it does override, i believe we should be good with this change,